.gitignore

```
@@ -33,6 +33,10 @@ build/
 local.properties
 *.iml
 
+# VS Code
+#
+.vscode/settings.json
```

```suggestion
.vscode/
```

It might be worth ignoring all vscode settings. We do this for other SDKs - ignore editor config files so each developer can setup the IDE to their own preferences.